packages/dappmanager/src/daemons/stakerDbUpdate/index.ts

```diff
@@ -0,0 +1,59 @@
+import { eventBus } from "../../eventBus.js";
+import { mevBoostMainnet, mevBoostPrater, stakerPkgs } from "@dappnode/types";
+import * as db from "../../db/index.js";
+import { logger } from "ethers";
+
+async function removeStakerPkgsFromDbIfSelected({
+  dnpNames
+}: {
+  dnpNames?: string[];
+}): Promise<void> {
+  if (!dnpNames) return;
```

Comment:
this checker will not work as you expect. The `if(!...)` does not check for arrays with length 0. You must check for `if(dnpNames.length === 0)